practice7/MaybeTheBestCodeEver.cpp

```diff
@@ -0,0 +1,156 @@
+
+#include <iostream>
+#include <vector>
+#include <string>
+#include <algorithm>
+#include <cstdio>
+#include <filesystem>
+//#include <print>
+
+int gCount = 0;
+double gBiggestArea = 0;
+double gSmallestArea = 1000000;
+
+class Rectangle {
+public:
+    double Width;
+    double Height;
+    std::string Name;
+
+    Rectangle(const std::string& n, double w, double h) : Name(n), Width(w), Height(h) {}
+};
+
+// Функція для створення вектора об'єктів Rectangle
+std::vector<Rectangle> createRectangles(int count) {
+    std::vector<Rectangle> rectangles;
+
+    for (int i = 0; i < count; ++i) {
+        double width, height;
+        std::string name = "Rectangle " + std::to_string(i + 1);
+        //std::println("Enter width and height for {}  : ", name());
```

Comment:
lets use println instead of cout:
```
for (int i = 0; i < count; i++) {
    auto name = std::format("Rectangle {}", i + 1);
    std::println("Enter width and height for {} : ",  name);

    double width, height;
    std::cin >> width >> height;
    rectangles.emplace_back(name, width, height);
}
```